dataland-frontend/tests/e2e/utils/GeneralUtils.ts

```diff
@@ -118,3 +118,26 @@ function checkIfContentIsIdentical(
     throwErrorBecauseOfFieldValue(newPath);
   }
 }
+
+/**
+ * This function opens a given dataset with its Multi Layer Data Table and checks wether a specific field can be hidden using the input switch.
+ * @param datasetIdentifier Identifies a dataset on the "my dataset" Page. The function will open the corresponding dataset.
+ * @param toggledFieldName Name of a field which is toggled by the input switch
+ */
+export function checkToggleEmptyFieldsSwitch(datasetIdentifier: string, toggledFieldName: string): void {
```

Comment:
The function name does not reflect that it visits the dataset view page first starting at the MyDatasets page. I feel like we should pull that out of this function.